src/models.py

```diff
@@ -496,11 +494,12 @@ def build_image_sent_module(task, d_inp, params):
     return pooler
 
 
-def build_single_sentence_module(task, d_inp, params):
+def build_single_sentence_module(task, d_inp, use_bert, params):
```

Comment:
Add a comment somewhere explaining why there isn't a parallel use_openai_transformer.

Reply:
Ideally we could just use a common interface to both, see #491

Reply:
For OpenAI transformer, the last hidden state is used? We could pass the entire model, as we do in `build_pair_sentence_module`.